pkgs/by-name/gr/grafana-to-ntfy/package.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  stdenv,
+  rustPlatform,
+  fetchFromGitHub,
+  pkgs,
+  ...
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "grafana-to-ntfy";
+  version = "0.1.0";
```

There doesn't seem to be any sort of stable releases upstream, and 0.1.0 seems to be a moving target. I think this versioning would be more reasonable:

```suggestion
  version = "0-unstable-2024-10-04";
```